### PR TITLE
Uplinks Get Dropped (SBO44)

### LINKED ISSUE #21528: Removing Someone's Tot Status Doesn't Automatically Remove Their Uplink
Easy fix, Drop() should call take_uplink()

### PR DESCRIPTION
**Pull request by Kurfursten**
fixes #21528

🆑 
* bugfix: Fixed a bug where someone who was removed as traitor would not have their uplink taken automatically.

### CODE REVIEW

File: code/datums/gamemode/role/syndicate.dm [hunk 1]
```diff
@@ -24,12 +24,14 @@
...
 		S.laws.zeroth_lock = FALSE
 		to_chat(S, "Law 0 has been purged.")
-	if(isAI(antag.current))
+	else if(isAI(antag.current))
 		var/mob/living/silicon/ai/KAI = antag.current
 		to_chat(KAI, "<b>Your laws have been changed!</b>")
 		KAI.set_zeroth_law("","")
 		KAI.laws.zeroth_lock = FALSE
 		KAI.notify_slaved()
+	else if(ishuman(antag.current))
```
Comment (jknpj) [inline code/datums/gamemode/role/syndicate.dm:33]: doesn't this mean that borged humans can keep their uplink?

Comment (Kurfursten) [reply code/datums/gamemode/role/syndicate.dm:33]: take_uplink is not a very intelligent proc, it just searches for an uplink anywhere on the person, so take_uplink would not do anything for borged humans anyway (there is no conceivable way they could be holding an object containing an uplink)

Comment (jknpj) [reply code/datums/gamemode/role/syndicate.dm:33]: It's a damn shame, because they can be made human again

Comment (Kurfursten) [reply code/datums/gamemode/role/syndicate.dm:33]: For the record, Drop() is only called by admins to remove someone's antag status, it doesn't happen when you get borged

**Pull request merged**